### DISCUSSION TITLE
Empty OPML, can't import

### DISCUSSION THREAD

**Discussion by nachocab** [mxpv/podsync#264] [General]
The XML file looks valid, but my podcast manager doesn't import it.

I also see an OPML file but it appears empty:

```
<?xml version="1.0" encoding="UTF-8"?>
<opml version="1.0">
	<head>
		<title>Podsync feeds</title>
	</head>
	<body></body>
</opml>
```

Am I missing something obvious?

**Comment by mxpv** [comment]
You'd need to set `opml = true` in your toml file in order to add a channel to opml index.

```
  # opml = true|false # Optional inclusion of the feed in the OPML file (default value: false)
```
